airbyte-cdk/java/airbyte-cdk/airbyte-commons/src/main/java/io/airbyte/commons/version/AirbyteProtocolVersion.java

```diff
@@ -6,8 +6,8 @@
 
 public class AirbyteProtocolVersion {
 
-  public final static Version DEFAULT_AIRBYTE_PROTOCOL_VERSION = new Version("0.2.0");
-  public final static Version V0 = new Version("0.3.0");
+  public final static Version DEFAULT_AIRBYTE_PROTOCOL_VERSION = new Version("0.5.0");
```

Comment:
Remind me again...why do we need to bump the version here?

Reply:
reverted